api/api.go

```diff
@@ -919,30 +919,32 @@ func (api *Server) getBlockMetas(start uint64, count uint64) (*iotexapi.GetBlock
 	if start > tipHeight {
 		return nil, status.Error(codes.InvalidArgument, "start height should not exceed tip height")
 	}
-	var res []*iotextypes.BlockMeta
+	var res []*iotextypes.BlockHeaderCore
 	for height := start; height <= tipHeight && count > 0; height++ {
-		blk, err := api.bc.GetBlockByHeight(height)
+		header, err := api.bc.BlockHeaderByHeight(height)
```

Comment:
Since there’s no numActions bellow the Bering height, you need to continue get the full block data instead